impl_core/src/main/java/io/opencensus/implcore/stats/MetricUtils.java

```diff
@@ -93,5 +85,34 @@ public Type apply(Aggregation arg) {
     return labelValues;
   }
 
+  private static final Function<Object, Type> TYPE_CUMULATIVE_DOUBLE_FUNCTION =
+      Functions.returnConstant(Type.CUMULATIVE_DOUBLE);
+
+  private static final Function<Object, Type> TYPE_CUMULATIVE_INT64_FUNCTION =
+      Functions.returnConstant(Type.CUMULATIVE_INT64);
+
+  private static final Function<Object, Type> TYPE_CUMULATIVE_DISTRIBUTION_FUNCTION =
+      Functions.returnConstant(Type.CUMULATIVE_DISTRIBUTION);
+
+  private static final Function<Object, Type> TYPE_GAUGE_DOUBLE_FUNCTION =
+      Functions.returnConstant(Type.GAUGE_DOUBLE);
+
+  private static final Function<Object, Type> TYPE_GAUGE_INT64_FUNCTION =
+      Functions.returnConstant(Type.GAUGE_INT64);
+
+  private static final Function<Object, Type> TYPE_UNRECOGNIZED_FUNCTION =
+      Functions.<Type>throwAssertionError();
```

Comment:
throwAssertionError already returns the same instance all the time, don't need to make it static.

Reply:
I see - this doesn't make a difference so I'll leave it as-is :)